### DISCUSSION TITLE
How to validate forms?

### DISCUSSION THREAD

**Discussion by Dave-Wagner** [akveo/react-native-ui-kitten#1517] [Q&A] [answered]
Has anyone integrated Formik with UI-Kitten, or any other form library? I need to validate input and I'd like the color of the input border to change based upon live validation. I've done this with Formik and NativeBase, FormControl made this pretty simple however, I can't find the way to do this with UI-Kitten.

**Comment by artyorsh** [comment, marked as answer]
Here is what I came to in production usage
https://gist.github.com/artyorsh/71a3aa498a9fbc32dec7b624bc764cf9

**Reply by Dave-Wagner** [reply]
Thanks for this example. I think I see now how I can implement it. However, if anyone has other examples please do add them here, in particular a React Native example.

**Reply by kushxogit** [reply]
Hey that page is deleted, please let me know how we are doing it?
